test/jdk/javax/swing/JFrame/JFrameBackgroundRefreshTest.java

```diff
@@ -136,17 +137,23 @@ private static void moveMouseSlowly( int targetX, int targetY) {
     }
 
     private static boolean compareImages(BufferedImage bufferedImage) {
-        int sampleRGB = bufferedImage.getRGB(0,0);
+        Color firstPixel = new Color(bufferedImage.getRGB(0,0));
         for (int x = 0; x < bufferedImage.getWidth(); x++) {
             for (int y = 0; y < bufferedImage.getHeight(); y++) {
-                if (bufferedImage.getRGB(x, y) != sampleRGB) {
+                if (!compareColor(firstPixel, new Color(bufferedImage.getRGB(x, y)))) {
                     return false;
                 }
             }
         }
         return true;
     }
 
+    private static boolean compareColor(Color c1, Color c2) {
```

Comment:
Shall we move such a method into `Util`?

More and more test cases add tolerance to color comparison. Yeah, it's simple enough… and adding a library makes it somewhat harder to run a test in stand-alone mode, without jtreg.

Reply:
Yeah, adding it to library will make it harder to run a test in stand-alone mode. There is this `Util.testBoundsColor()` in Util and I don't see much of test using it though. I guess it's better to leave the comparison to individual test, rather than trying to make it generic.